app/src/main/java/st235/github/com/catovasia/MainActivity.java

```diff
@@ -5,12 +5,49 @@
 
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import st235.github.com.catovasia.data.DataCallback;
+import st235.github.com.catovasia.data.NetRepository;
+import st235.github.com.catovasia.data.ResourceRepository;
+import st235.github.com.catovasia.models.PictureResponse;
 
 public class MainActivity extends AppCompatActivity {
 
+    private static String TAG = "[Catovasia] MainActivity";
+
+    @Inject
+    NetRepository netRepository;
```

Comment:
Repositories in activity? Hmm

Reply:
firstly, it looks like you're incompetent on architecture :disappointed: .
why its bad?
what do you think what the architecture my project follows?

Reply:
Well, we'll see what we can do with it :smirk_cat: